src/components/VirtualizedTreeSelect.js

```diff
@@ -352,10 +352,14 @@ const Menu = (props) => {
           props.selectProps.listProps.onScroll(e.target);
         },
         //Enables option selection even when input is not focused
-        onMouseDown: (event) => {
+        onMouseEnter: (event) => {
```

Comment:
Isn't this a bit too eager? Mouse enter will trigger focus whenever mouse enters the menu list, which could mean it will steal focus from other components accidentally. How does this help with the keyboard navigation?

Reply:
Yes I agree that it is quite eager, if you think it makes sense I would remove this custom Menu (see reply above), otherwise I will think about another implementation.

Reply:
Let's remove the component in that case.